src/services/webdav/backend.rs

```diff
@@ -15,10 +15,12 @@
 use std::collections::HashMap;
 use std::fmt::Debug;
 use std::fmt::Formatter;
+use std::path::PathBuf;
```

Comment:
Pleasse don't use `PathBuf` in a http based services.

Reply:
How to normalize path without `PathBuf`? Or we don't need to normalize path?

Reply:
All input pathes are already normalized.

Reply:
Cool, thanks!